.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: '[feature]'
+labels: feature
+assignees: ''
+
+---
+
+### **Is your feature request related to a problem? Please describe.**
```

Comment:
These can be unbolded now that they're all headings. They'll be larger and bold by default

Reply:
In the other file too

Reply:
Should the pull request template also use ### for headings (for consistency)?

Reply:
Yeah, we should choose one or the other